styleguide/_themes/derek/scss/components/mainnav.scss

```diff
@@ -0,0 +1,371 @@
+//main nav
+.main-nav {
+  background: $dark-gray;
+  border: 0;
+  border-radius: 0;
+  margin: 0;
+
+  .navbar-menuContainer {
+    border: 0;
+    max-height: none;
+    padding: 0;
+  }
+
+}
+
+.navbar-nav {
+
+  .navbar-topItem {
+    font-size: 1.3rem;
+    font-weight: 400;
+    text-transform: uppercase;
+
+    .navbar-topLink {
+      color: $white;
+      display: block;
+      padding: 21px 15px;
+      text-decoration: none;
+
+      &:hover,
+      &:focus,
+      &:active,
+      &:visited {
+        background: $medium-dark-gray;
+      }
+
+    }
+
+    &.open {
+      > .navbar-topLink {
+        background: $medium-dark-gray;
+      }
+    }
+
+    .navbar-dropDownLink {
+      color: $white;
+      font-size: .8em;
+      font-style: italic;
+      padding: 0;
+      text-decoration: none;
+      text-transform: none;
+      padding: 5px 20px;
+
+      &:hover,
+      &:visited,
+      &:focus {
+        background-color: transparent;
+      }
+
+    }
+  }
+}
+
+.navbar-dropDown-trigger {
+
+  &::after {
+    content: '\f0da';
+    float: right;
+    font-family: 'FontAwesome';
+  }
+
+}
+
+.navbar-container {
+  padding: 0;
+  margin: 0 auto;
+  max-width: 960px;
+}
+
+.navbar-footer {
+  bottom: 0;
+  position: fixed;
+  width: 100%;
+}
+
+.navbar-footerLink {
+  @extend h6;
```

Comment:
This will obviously get picked up by the sass linter, but all @extends should use placeholders instead of classes, use a mixin instead, or just place the properties directly into the selector.